shared/bundle_analysis/report.py

```diff
@@ -239,12 +239,14 @@ def ingest(self, path: str, compare_sha: Optional[str] = None) -> Tuple[int, str
 
             # Save custom base commit SHA for doing comparisons if available
             if compare_sha:
```

Comment:
This needs to be an upsert, this is one of the only ways of doing it for SQLAlchemy/SQLite